### DISCUSSION TITLE
How to deactive Inert attribute added by dropdown in Portal

### DISCUSSION THREAD

**Discussion by ChloeAidi** [tailwindlabs/headlessui#3256] [Help]
Hello,
In headless ui 2, the combobox dropdown is opened in a portal container. This is very convenient, however, it also seems to add the inert attribute on all the other elements.

![image](https://github.com/tailwindlabs/headlessui/assets/122309239/33ab5fb6-1a5a-4584-90e9-cc70ca003bb3)

I am using a button to remove the user input ( the 'x' button) . This is a sibling of the combobox input. 
```
<ComboboxInput
	[...]
/>
<ClearButton />
```
This button get the attribute 'inert' when the dropdown is opened.
Is there a way to overwrite or remove this attribute so the user can clear its input when the dropdown is opened?

**Comment by Amitojasa** [comment]
I am facing similar issue I am also not able to select any Item from dropdown

**Comment by y04nqt** [comment]
https://github.com/tailwindlabs/headlessui/issues/3421 I created an issue around the same problem I'm facing using Combobox.
